guides/v2.1/frontend-dev-guide/css-topics/css-preprocess.md

```diff
@@ -63,7 +65,7 @@ In the Magento application, the following modes of compiling <code>.less</code>
 
 To set the compilation mode, do the following:
 
-1.  In the Magento Admin, navigate to **Stores** \> **Configuration** \> **ADVANCED** > **Developer**.
+1.  In the Magento Admin, navigate to **Stores** \> **Configuration** \> **Advanced** > **Developer**.
```

Comment:
There are two extra `/` here for some reason. I'll remove them, but make sure you check for inconsistencies like this!